tools/grama.sed

```diff
@@ -1,6 +0,0 @@
-# remove index "see" entries
-/\\index.*|see/d
```

Comment:
Is this unnecessary now? (How? Do we simply not have any of these in extracted groups?)